packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -62,20 +64,58 @@ Stacked bar charts describe totals while allowing a degree of internal breakdown
+These are the most common use cases for donut charts:
+  - Showing the relationship of a set of values to a whole
+  - Showing progress completion as a percentage
+  - Showing utilization for an object as a percentage (with or without thresholds)
 
-#### Example
-![Donut chart](./img/donutchart.png)
+Donut charts can use percentages or whole numbers to compare parts to the whole. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another. Decide whether percentages or whole numbers is most compatible with your use case and make sure to be consistent throughout. We recommend displaying these values within the donut, see [Donut utilization](#donut-utilization) for styling. Don’t use more than six categories in total. We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) guidelines to represent your data when thresholds are not present.
+
+![Donut chart](./img/donut chart.png)
+
+1. **Segment fill**: We recommend using [Colors for charts] (/design-guidelines/styles/chart-colors) for different items within the donut chart.
+2. **Segment padding**:  Always provide 3px of padding between segments.
+3. **Chart tooltip**:  A tooltip will appear upon hover that states the segment name and value. Depending on the type of donut, the information may change.
+  - **Donut**: We recommend stating the segment name and the value being represented. For example, if the segment represents “Bugs,” and the value being represented is 25, your chart tooltip would state, “Bugs - 25.”
+  - **Donut utilization**: Only the utilization segment will allow for hover. We recommend stating the segment name and the utilization value being captured. For example, if the user is tracking GBps utilization, the chart tooltip would state “GBps utilization - 75%.”
+  - **Donut utilization with thresholds**: We recommend stating the threshold name, the segment name, and the value being represented. For example, if the threshold being met is “Danger,” and the segment represents “Storage capacity,” and the value being represented is 92%, your chart tooltip would state, “Danger: Storage capacity - 92%.”  We recommend stating the threshold name “at” the percentage the threshold begins (ex. Warning at 60%).
+4. **Legend**: In order to fulfill a11y compliance requirements, we recommend adding a legend to all donuts.
+
+<!--#### React component
+[Donut chart](/documentation/react/charts/donut)-->
+
+### Donut utilization
+![Donut utilization](./img/donut utilization.png) ![Donut utilization hover](./img/donut utilization hover.png)
```

Comment:
These images are not displaying in preview. Think it's the spaces in file names.